client/src/components/User/UserContext/index.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from "react";
```

Comment:
Would be a good idea to move this file to a context folder rather than the components folder for clarity